docs/docs/traditional-ml/hyperparameter-tuning-with-child-runs/notebooks/parent-child-runs.ipynb

```diff
@@ -368,14 +368,6 @@
     "    )\n",
     ")"
    ]
-  },
```

Comment:
This is unrelated change to remove unnecessary blank cell in https://mlflow.org/docs/latest/traditional-ml/hyperparameter-tuning-with-child-runs/notebooks/parent-child-runs

Reply:
Non blocking, but let's avoid making unrelated updates :)

Reply:
Let me separate PRs then 😄

Reply:
This looks helpful, let me file a separate PR including the fix and linter update